src/pages/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,118 @@
-import React from "react"
+import React, { useState } from "react"
+
+import styles from "./Gallery.module.scss"
+import classnames from "classnames"
+
+const Lines = () => {
+  return (
+    <div className={styles.lines}>
+      {Array.from({ length: 6 }).map((_, index) => (
+        <div key={index} className={styles.line} />
+      ))}
+    </div>
+  )
+}
 
 const Gallery = () => {
-  return <h1>Wojak gallery here</h1>
+  const [isFilterOpen, setIsFilterOpen] = useState(false)
+
+  return (
+    <>
+      <div>
+        <div className={styles.pageContainer}>
```

Comment:
Take a look at `App/components/Layout` component, I think you can adjust styles there so we'll make them global for every page and remove this wrapper.